src/Uno.Extensions.Reactive.Generator/Utils/StringExtensions.cs

```diff
@@ -109,4 +110,7 @@ public static string Replace(this string text, char[] chars, char replacement)
 
 		return result.ToString();
 	}
+
+	public static string RemoveEmptyLines(this string text)
+		=> Regex.Replace(text, "^\\s*$", string.Empty, RegexOptions.Multiline);
```

Comment:
Would benefit from being a static compiled member.

Reply:
This was not used, removed